test/e2e/multicast_test.go

```diff
@@ -600,25 +600,57 @@ func runTestMulticastBetweenPods(t *testing.T, data *TestData, mc multicastTestc
 		data.RunCommandFromPod(data.testNamespace, senderName, mcjoinContainerName, sendMulticastCommand)
 	}()
 
+	runIpCmd := func(nodeName string, cmd []string, outputFilters ...string) ([]string, error) {
```

Comment:
I think this function doesn't have to be used with an IP command. Maybe change its name to `runIPCmdWithFilters.`